### PR TITLE
Setup Dev Environment

### PR DESCRIPTION
**Pull request by akoushke**
ATM, `eslint` does not lint the scripts and I don't mind that. I'm assuming this `start.js` file will never be committed again.

### CODE REVIEW

File: scripts/start.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+if (process.env.WEBEX_ACCESS_TOKEN) {
+  const webex = new Webex({
+    credentials: process.env.WEBEX_ACCESS_TOKEN,
+  });
+
+  // eslint-disable-next-line no-unused-vars
+  const webexSDKAdapter = new WebexSDKAdapter(webex);
+} else {
+  // eslint-disable-next-line no-console
+  console.error('Please set a valid WEBEX_ACCESS_TOKEN environment in `env.development` file at the root level.');
```
Comment (adamweeks) [inline scripts/start.js]: Can we `return process.exit(1);` here so the node script knows it failed?

Comment (akoushke) [reply scripts/start.js]: then I could get rid of the `else` condition. just have a quick `if` statement at top.

File: scripts/start.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import * as dotenv from 'dotenv';
+import Webex from 'webex';
+
+import WebexSDKAdapter from '../src/WebexSDKAdapter';
+
+// Allows convict module to read the corresponding configuration
+dotenv.config({path: `${__dirname}/../.env.development`});
```
Comment (adamweeks) [inline scripts/start.js]: Can we just have the file loaded from `.env` and then all we have to do is `dotenv.config()`

File: scripts/start.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import * as dotenv from 'dotenv';
+import Webex from 'webex';
+
+import WebexSDKAdapter from '../src/WebexSDKAdapter';
+
+// Allows convict module to read the corresponding configuration
```
Comment (adamweeks) [inline scripts/start.js]: Not sure what "convict" means.

File: scripts/start.js (hunk 2 above)
Comment (akoushke) [reply scripts/start.js]: I liked having a dev environment as a separate environment to not mess things up later on, in case we need actual env variables.

Comment (akoushke) [reply scripts/start.js]: I can change it to what you want, if you still think it's not necessary.

File: package.json [hunk 4]
```diff
@@ -25,15 +26,18 @@
   "author": "devsupport@webex.com",
   "license": "MIT",
   "dependencies": {
-    "@webex/components": "^1.18.1",
+    "@webex/component-adapter-interfaces": "^1.2.1",
```
Comment (lalli-flores) [inline package.json]: Can you have the install/remove from components repot to interface repo on its own commit?

File: package.json [hunk 5]
```diff
@@ -25,15 +26,18 @@
...
+    "dotenv": "^8.2.0",
```
Comment (lalli-flores) [inline package.json]: This is a dev dependency

File: package.json [hunk 6]
```diff
@@ -25,15 +26,18 @@
...
     "rxjs": "^6.5.3",
     "webex": "^1.79.0"
```
Comment (lalli-flores) [inline package.json]: We should make `webex` a peer dependency

File: scripts/start.js [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+
+  process.exit(1);
+}
+
+const webex = new Webex({
+  credentials: process.env.WEBEX_ACCESS_TOKEN,
+});
+
+// eslint-disable-next-line no-unused-vars
+const webexSDKAdapter = new WebexSDKAdapter(webex);
```
Comment (lalli-flores) [inline scripts/start.js]: Can you add a comment after this? something like

```
--------------------------------------------------------------------------------
// Add your test/development code here

File: package.json (hunk 6 above)
Comment (akoushke) [reply package.json]: this can not be a peer dependency in this repo. we are utilizing it in our scripts now.

Comment (akoushke) [reply package.json]: I will change it a `devDep`

Comment (lalli-flores) [reply package.json]: I'm ok with that, we don't want to bundle our implementation with the SDK.

File: scripts/start.js (hunk 3 above)
Comment (lalli-flores) [reply scripts/start.js]: It's a [tool](https://www.npmjs.com/package/convict) we used in previous projects, prob copy-paste issue

**Pull request merged**